spec/comment_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject do
+    user = User.new(name: 'Javier', photo: 'https://whereever.com', bio: 'Surfer', posts_counter: 0)
+    Post.create(title: 'Good Habits', text: 'This is why habits are necessary.', comments_counter: 4,
+                likes_counter: 6, user:)
+  end
+
```

Comment:
- [ ] [REQUIRED] Great job so far but we have failed tests because of a syntax error.
<img width="826" alt="Screenshot 2023-04-09 at 02 08 28" src="https://user-images.githubusercontent.com/58692590/230749167-8d586230-7792-4a82-9442-ac2199745380.png">

The problem is that you are passing an empty hash as the value for the user parameter. This is causing a syntax error because the ) character after user: is unexpected.

To fix the error, you need to pass a valid value for the user parameter. This should be a reference to a User instance that you have created in your test.

Here's an example of how you could modify your code to fix the error:

``` ruby
RSpec.describe Comment, type: :model do
  subject do
    user = User.create(name: 'Javier', photo: 'https://whereever.com', bio: 'Surfer', posts_counter: 0)
    Post.create(title: 'Good Habits', text: 'This is why habits are necessary.', comments_counter: 4, likes_counter: 6, user: user)
  end

  it 'Comments counter should equal 4' do
    expect(subject.comments_counter).to eq(4)
  end
end

```